src/test/java/com/google/api/codegen/gapic/testdata/py/python_library.baseline

```diff
@@ -4153,7 +4153,11 @@ def sample_discuss_book():
   client = library_v1.LibraryServiceClient()
 
   name = 'BASIC'
-  request = {'name': name}
+  file_name = os.path.join(os.path.dirname(__file__), 'comment_file')
```

Comment:
as clarified at the meeting, don't require `dirname(__file__)`. Assume a dir relative to the cwd, or an absolute path if it starts with `/`